src/cat-data-service/src/poem_types/registration.rs

```diff
@@ -0,0 +1,75 @@
+use chrono::{DateTime, Utc};
```

Comment:
I agree with your point, but right now it is not possible easily to fix it because we are using chrono crate in event-db, so cat-data-service should has corresponded version of chrono based on poem and event-db. Need to think how it is better to solve this problem.